symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/config/BdkExtAppControllerConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.symphony.bdk.app.spring.config;
+
+import com.symphony.bdk.app.spring.SymphonyBdkAppProperties;
+import com.symphony.bdk.app.spring.auth.CircleOfTrustController;
+import com.symphony.bdk.app.spring.exception.GlobalControllerExceptionHandler;
+import com.symphony.bdk.core.auth.ExtensionAppAuthenticator;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Configuration and injection of the main rest controllers for extension app APIs as beans within the Spring application context.
+ */
+public class BdkExtAppControllerConfig {
+
+  @Bean
+  @ConditionalOnProperty(name = "bdk.app.auth.enabled", havingValue = "true")
+  public CircleOfTrustController circleOfTrustController(SymphonyBdkAppProperties properties,
```

Comment:
Since `CircleOfTrustController` is annotated by `@RestController`, are we sure that it's not injected twice in the spring application context?

Reply:
when I set `bdk.app.auth.enabled=false`, the Api returns 404 Not Found, so I guess the controller is not injected twice